style.go

```diff
@@ -183,7 +183,8 @@ func DefaultFill() *Fill {
 }
 
 func DefaultBorder() *Border {
-	return NewBorder("none", "none", "none", "none")
+	//return NewBorder("none", "none", "none", "none")
+	return NewBorder("", "", "", "")
```

Comment:
Excel unfortunately isn't terribly robust, and files that work perfectly well with LibreOffice or Google Sheets may not look quite right. I was having trouble getting the borders to appear, so I was trying to bring the output of this library more in line with what Excel provides, in the hopes that it would be happier. And one of the differences is that Excel doesn't use "none", it uses empty quotes.

It turns out that this wasn't the problem with this particular function -- the problem turns out to be that Excel is expecting a set of borders to exist that are never used - it wants a set of empty borders as the first set, even if no style ever references them.